cypress/integration/default/images.spec.ts

```diff
@@ -20,24 +19,20 @@ describe('next/images', () => {
 
   it('should show image allow-listed with remotePatterns', () => {
     cy.visit('/image')
-    cy.findByRole('img',{ name: /tawny frogmouth/i }).should('be.visible').and(($img) => {
-      // "naturalWidth" and "naturalHeight" are set when the image loads
-      expect(
-        $img[0].naturalWidth,
-        'image has natural width'
-      ).to.be.greaterThan(0)
-    })
+    cy.findByRole('img', { name: /tawny frogmouth/i })
+      .should('be.visible')
+      .and(($img) => {
+        // "naturalWidth" and "naturalHeight" are set when the image loads
+        expect($img[0].naturalWidth, 'image has natural width').to.be.greaterThan(0)
+      })
   })
 
-  it('should show a broken image if it is not on domains or remotePatterns allowlist', () => {
-    cy.visit('/image')
-    cy.findByRole('img',{ name: /jellybeans/i }).should('be.visible').and(($img) => {
-      expect(
-        $img[0].style.height
-      ).to.equal('0px')
-      expect(
-        $img[0].style.width
-      ).to.equal('0px')
+  it('should show throw if an image is not on the domains or remotePatterns allowlist', () => {
+    cy.request('/broken-image').then((response) => {
+      expect(response.status).to.eq(200)
```

Comment:
The image is broken, but the page still loads with the error message which is why this is an HTTTP 200